### ISSUE TITLE
[Innspill til komponent]: <HelpText> vis ved hover

### ISSUE THREAD

**Issue by ivoiv** [navikt/Designsystemet#2779]
### Tilbakemelding

I dag vises HelpText kun ved trykk.
Det hadde vært fint om det også støttet mulighet for å vises når man holder musepekeren over den. 

Det blir som Tooltip, bare med en tekstboks som er mer passelig for større tekster.

**Comment by KenAJoh** [comment]
Hei 👋 Vi tenker dette kanskje vil være en egen komponent, da vi ikke ønsker å ha flere metoder for å interagere med samme komponent på tvers av flater. 
Har dere en løsning i dev eller noen screenshots vi kunne brukt som inspirasjon for hvordan dette mønsteret brukes på ulike flater? Vi ser et behov for en lignende komponent, spesielt for interne systemer så alle eksempler hjelper 🙌

**Comment by ivoiv** [comment]
Dette var vi i dag i en litt eldre tjeneste. 
Så ønsker noe likt, men med mulighet for å holde Popover åpent dersom man flytter musepekeren til den, for å kunne kopiere teksten.

Det fungerer naturligvis ikke på mobil/tablet så der må man trykke på knappen i stedet. 

(Sorry, skjønner ikke hvorfor GIFen ble slow-motion)

![helptekst](https://github.com/navikt/aksel/assets/11502345/512868ed-d5a3-4d52-b475-1a7666fa520a)

**Comment by sjur-gr** [comment]
Som Ken skriver så ønsker vi ikke å legge opp til at UX'en blir forskjellig på tvers av systemene. At noen steder er det hover og andre steder er det klikk. Begge deler blir knotete UX. Jeg løfter spm ut i kanelen for interne flater og hører om det er noen utfordringer med å endre til hover som standard.

**Comment by ivoiv** [comment]
@sjur-gr 
Evt. legge til rette for custom styling eller forbedring av Tooltip tekstboksen?
Den er veldig liten med lite padding og liten tekst, så det blir uoversiktlig med lengre tekster.

![image](https://github.com/navikt/aksel/assets/11502345/d3b357f4-5501-443b-9c34-04dd7e32f5fd)

**Comment by sjur-gr** [comment]
Ja, det er by design. Tooltip brukes kun til å beskrive en handling.

**Comment by KenAJoh** [comment]
Vi har ikke sett behovet for denne interaksjonen eller det siste året så vi har ikke på planene å implementere dette lokalt. Men skal være ganske rett-frem å bruke floating-ui for "hover"-interaksjon.
https://floating-ui.com/docs/tooltip

Men er en del uu-regler som må støttes for hover/fokus
- https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html
- https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/

> The intent of this condition is to ensure that additional content which may appear on hover of a target may also be hovered itself. Content which appears on hover can be difficult or impossible to perceive if a user is required to keep their mouse pointer over the trigger. When the added content is large, magnified views may mean that the user needs to scroll or pan to completely view it, which is impossible unless the user is able to move their pointer off the trigger without the additional content disappearing.
